examples/navier-stokes/navierstokes.c

```diff
@@ -660,8 +660,8 @@ int main(int argc, char **argv) {
   PetscFunctionListFind(qflist, problemtype, &fp);
   if (!fp)
       return CeedError(ceed, 1, "Function not found in the list");
-  strcpy(str, __FILE__":");
-  strcat(str, problemtype);
+  PetscStrncpy(str, __FILE__":", 12);
```

Comment:
The constant `__FILE__` is likely to be longer than 12 characters

Reply:
Good point, I'll switch it to 256.